pkg/controller/storage/container/container.go

```diff
@@ -90,10 +90,10 @@ func Setup(mgr ctrl.Manager, l logging.Logger) error {
 
 // Reconcile reads that state of the cluster for a Provider acct and makes changes based on the state read
 // and what is in the Provider.Spec
-func (r *Reconciler) Reconcile(request reconcile.Request) (reconcile.Result, error) {
+func (r *Reconciler) Reconcile(ctx context.Context, request reconcile.Request) (reconcile.Result, error) {
```

Comment:
Oy, I forgot we still had this controller that doesn't use `managed.Reconciler`. :(